app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -57,17 +57,19 @@ def add_members(new_member_ids)
 
     # Add member objects by adding this collection to the objects' member_of_collection association.
     def add_member_objects(new_member_ids)
+      error_messages = []
       Array(new_member_ids).each do |member_id|
         member = ActiveFedora::Base.find(member_id)
-        # @note Ideally, this would be surfaced as a warning in a flash
-        #       message. Because the member is found and saved in this model
-        #       method, I am not sure it's worth the effort to rejigger things
-        #       such that this information bubbles up to the controller and
-        #       view.
-        next if Hyrax::MultipleMembershipChecker.new(item: member).check(collection_ids: id, include_current_members: true)
+        # will return and error message as a string, otherwise it will return a nil
```

Comment:
See prior code review for suggestions around this file. (They were auto-hidden by GitHub prematurely.)